curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/map-the-debris.md

```diff
@@ -8,12 +8,23 @@ dashedName: map-the-debris
 
 # --description--
 
+The orbital period is the amount of time a given astronomical object takes to complete one orbit around another object.
+
+According to Kepler's Third Law, the orbital period T of two point masses orbiting each other in a circular or elliptic orbit is:
+
+$$
+T = 2 \pi \sqrt{\frac{a^{3}}{\mu}}
+$$
+
+- a is the orbit's semi-major axis
+- μ = GM is the standard gravitational parameter
+- G is the gravitational constant,
+- M is the mass of the more massive body.
```

Comment:
```suggestion
According to Kepler's Third Law, the orbital period $T$ of two point masses orbiting each other in a circular or elliptic orbit is:

$$
T = 2 \pi \sqrt{\frac{a^{3}}{\mu}}
$$

- $a$ is the orbit's semi-major axis
- $μ = GM$ is the standard gravitational parameter
- $G$ is the gravitational constant,
- $M$ is the mass of the more massive body.
```
We should wrap the variables in inline mathjax both for formatting and to facilitate translation.